circus-train-integration-tests/src/test/resources/test-log4j.xml

```diff
@@ -32,6 +32,10 @@
     <level value="DEBUG" />
   </logger>
 
```

Comment:
Why is this added?

Reply:
Travis limits log output to 4MB, some of the recent changes in master have pushed the output above this which then fails the Travis build. It's not directly related to this PR but I needed to do this in order to get Travis passing again. We're probably going to have to do a bit of log output tuning in future whenever we hit this which isn't ideal but hey.